kubeflow/common/prototypes/ambassador.jsonnet

```diff
@@ -5,6 +5,7 @@
 // @param name string Name
 // @optionalParam platform string none supported platforms {none|gke|minikube}
 // @optionalParam ambassadorServiceType string ClusterIP The service type for the API Gateway.
+// @optionalParam ambassadorNodePort number 30080 The port number to expose ambassador on when ambassadorServiceType is NodePort
```

Comment:
Thanks for the PR!
Can you change default value of ambassadorNodePort to null and only add
```
nodePort: params.ambassadorNodePort
```
When ambassadorServiceType is NodePort and ambassadorNodePort is not null?

Reply:
Hi @kunmingg, I've implemented the change with the conditions described above. I kept the ambassadorNodePort parameter as a number with default value 0. If the ambassadorServiceType is changed to NodePort, by default, a random available port will be picked by k8s. The nodePort will be included in the configuration only if the parameter is set to a valid number from the range.